vcpkg.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "kdutils",
+    "version": "0.1.12",
+    "description": "KD Utilities Library - Core utilities and helper functions",
+    "homepage": "https://github.com/KDAB/kdutils",
+    "license": "MIT",
+    "supports": "!(uwp | arm)",
```

Comment:
!arm is a bit sus

Reply:
Yup, overlooked it. Now I made it more truthful.